packages/blade/src/components/Input/BaseInput/BaseInput.tsx

```diff
@@ -58,6 +59,14 @@ export type BaseInputProps = FormInputLabelProps &
      * The callback function to be invoked when the value of the input field changes
      */
     onChange?: FormInputOnEvent;
+    /**
+     * The callback function to be invoked when the value of the input field has any input
+     */
+    onInput?: FormInputOnEvent;
```

Comment:
`onChange` only triggers when actual value changes but I need to move from one input to another even when the actual value remains the same. For instance, if value of otpInput1 is `7` and I select the value and press `7` again, `onChange` won't be triggered since there is no change but I still need to change the focus to the next input.